04_014/app/server/src/ssr.service.js

```diff
@@ -0,0 +1,37 @@
+const { SSR } = require('../public/ssr.js')
+
+const getStringHtml = async (context) => {
+
+  const { state } = context
+
+  const ssrRendered = await SSR.ssr_converter(context);
```

Comment:
이 코드는 오류 때문에 제가 수정한건데, 원래 이렇게 되어있었습니다.
```suggestion
  const ssrRendered = await SSR.ssr_converter(context).then(result);
```
이 때 result가 해당 line에 없는 상태라서 오류가 발생한답니다.
그리고 저기서 then의 인자는 함수가 되어야 합니다. result는 함수가 아니라 단순한 값으로 사용되는 느낌이라서 사용 방법 자체가 잘못된 형태입니다.